CHANGES.rst

```diff
@@ -7,6 +7,7 @@ Changelog
 
 - Fix in template objects [nzambello]
 - Fix js event handling [nzambello]
+- Gallery view: added support for collections of images [nzambello]
```

Comment:
This is ok. I would release next version as 1.0.0 if it's woking correctly with collections and folders and now that #8 is fixed. @cekk what do you think?